jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/DockerClientParameters.java

```diff
@@ -48,6 +49,9 @@ public String getExecutable() {
   @Internal
   @Nullable
   Path getExecutablePath() {
+    if (System.getProperty(PropertyNames.DOCKER_CLIENT_EXECUTABLE) != null) {
```

Comment:
I think we should double-check if this is needed. I feel like `getExecutable()` *might* set `executable` when the property is defined.

Reply:
It doesn't look like it does.